decidim-initiatives/spec/system/create_initiative_spec.rb

```diff
@@ -631,23 +652,15 @@
             end
           end
 
-          context "when the scope is not selected" do
```

Comment:
This spec needs to be removed. The Initiative relies on the scoped types, which are being validated on creation, as part of the associations. 
Therefore this is leading to a *change* in the behavior. The CreateIntitiative command will use the first available `InitiativesTypeScope` record. Therefore, the fill in step will automatically select the first available scope. 

To avoid this, we need to move the scope selection  in the start section.